output/plugins/common/xml/common.cppcode

```diff
@@ -86,8 +86,9 @@ Written by
   <templates class="wxStaticText">
     <template name="declaration">#class* $name;</template>
     <template name="include">@#include &lt;wx/stattext.h&gt;</template>
-    <template name="construction">$name = new #class( #wxparent $name, $id, $label, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, $window_name @} );</template>
-  	<template name="settings">$name->Wrap( $wrap );</template>
+    <template name="construction">$name = new #class( #wxparent $name, $id, wxEmptyString, $pos, $size, $style #ifnotnull $window_style @{ |$window_style @} #ifnotnull $window_name @{, $window_name @} );</template>
```

Comment:
To reduce the code needed when not using markup, I would still pass the label to the constructor here, when `$markup` doesn't equal 1. Ideally the output shouldn't change at all when updating wxFormBuilder without changing a setting.

Reply:
OK